fairroot/base/steer/FairRootManager.h

```diff
@@ -63,12 +63,6 @@ class FairRootManager : public TObject
     0 : Branch does not exist   */
     Int_t CheckBranch(const char* BrName);
 
-    void CloseSink()
-    {
-        if (fSink) {
-            fSink->Close();
-        }
-    }
```

Comment:
Sadly enough this is "public" API. So we need to deprecate it as well before removing it…

Reply:
Sure.